1. Line 11 would output 3, assuming the use the values given to us in Question 4. This output would be possible because the valiable i is defined as var as therefore can be used globally (outside the loop).
2. Line 12 would output 150, assuming the use the values given to us in Question 4. This output would be possible because the valiable discountedPrice is defined as var as therefore can be used globally (outside the loop).
3. Line 12 would output 150, assuming the use the values given to us in Question 4. This output would be possible because the valiable finalPrice is defined as var as therefore can be used globally (outside the loop).
4. The function would output an array with values of [50, 100, 150]. The reason for the output being successful as there are no errors, all of the variables are global. And finally, the dicsount applied is 50% (accoridng to 0.5 being input) and the function outputs discounted prices.
5. An error would happen in line 11. The reason for that is because i is defined as variable using let, and that means that it could only be used in that loop. 
6. An error would happen in line 12. The reason for that is because dicountedPrice is defined as variable using let, and that means that it could only be used in that loop. 
7. Line 13 would output 150, assuming the use the values given to us in Question 8. This would be possible because finalValue is defined outside the loop, even though it is defined with let, it could be used thoughout the whole function.
8. The function would output the error, because for the use of the function we need i, and since it is defined with let, it would not exist outside the loop and it would be impossible to use it.
